tests/test_kubo_cas.py

```
@@ -0,0 +1,146 @@
+
+@pytest.mark.ipfs
+@pytest.mark.asyncio(loop_scope="session")
+@given(data=ipld_strategy())
+@settings(deadline=1000, print_blob=True)
+async def test_kubo_default_urls(global_client_session, data: IPLDKind):
+    """
+    Tests KuboCAS using its default URLs and when None is passed for URLs.
+    Requires a local IPFS daemon on default ports.
+    """
+    # Check if local IPFS daemon is available on default ports
+    import http.client
```

Pylint (3.3.7)

[convention] Import outside toplevel (http.client)

(C0415)